vnext/ReactUWP/Views/ShadowNodeBase.cpp

```diff
@@ -30,6 +31,13 @@ void ShadowNodeBase::updateProperties(const folly::dynamic &&props) {
 
 void ShadowNodeBase::createView() {
   m_view = GetViewManager()->CreateView(this->m_tag);
+
+  if (auto uielement = m_view.try_as<winrt::UIElement>()) {
+    if (g_HasActualSizeProperty == TriBit::Undefined) {
+      // ActualSize works on 19H1+ only
```

Comment:
swap the order these two happen in, no use calling QueryInterface if we already set the global variable